src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -211,6 +212,27 @@ def __init__(self, app, config, mount):
         identifier = cdict.get("log_reporter", "reqmgr2")
         self.logdb = LogDB(centralurl, identifier)
 
+        # local team cache which will request data from wmstats
+        self.wmstatsurl = cdict.get('wmstats_url', \
+                self.reqmgr_url.replace('reqmgr2', 'wmstatsserver'))
+        if  not self.wmstatsurl:
+            raise Exception('ReqMgr2 configuration file does not provide wmstats url')
+        self.team_cache = []
+
+    def getTeams(self):
+        "Helper function to get teams from wmstats or local cache"
+        teams = self.team_cache
+        try:
+            url = '%s/data/teams' % self.wmstatsurl
+            params = None
+            headers = {'Content-type':'application/json'}
+            data = getdata(url, params, headers)
+            teams = data.get('result', [])
+            self.team_cache = teams
+        except:
+            pass
```

Comment:
Seangchan,
I fixed issue with fetching teams content and added warning printouts.
Please review. The reqmgr2-dev is working now.
Valentin.

On  0, ticoann <notifications@github.com> wrote:
> > +        if  not self.wmstatsurl:
> > +            raise Exception('ReqMgr2 configuration file does not provide wmstats url')
> > +        self.team_cache = []
> > +
> > +    def getTeams(self):
> > +        "Helper function to get teams from wmstats or local cache"
> > +        teams = self.team_cache
> > +        try:
> > +            url = '%s/data/teams' % self.wmstatsurl
> > +            params = None
> > +            headers = {'Content-type':'application/json'}
> > +            data = getdata(url, params, headers)
> > +            teams = data.get('result', [])
> > +            self.team_cache = teams
> > +        except:
> > +            pass
> 
> May be we should pass some warning messages here
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6681/files#r55636476